modules/kms-keyring-browser/test/kms_keyring_browser.test.ts

```diff
@@ -51,46 +50,6 @@ describe('KmsKeyringBrowser::constructor', () => {
   })
 })
 
-describe('KmsKeyringBrowser can encrypt/decrypt with AWS SDK v2 client', () => {
-  const generatorKeyId =
-    'arn:aws:kms:us-west-2:658956600833:alias/EncryptDecrypt'
```

Comment:
Just because we are changing the default, why are we removing the tests for the old SDK?